README.md

```diff
@@ -91,6 +91,7 @@ For the experimental JupyterLab extension, install the Python package, make sure
 $ pip install bqplot
 $ jupyter labextension install @jupyter-widgets/jupyterlab-manager # install the Jupyter widgets extension
 $ jupyter labextension install bqplot
+$ jupyter lab build
```

Comment:
This should not be needed as `jupyter labextension install bqplot` did build Jupyter Lab. I wonder why it's needed in Binder, maybe they disabled building jupyterlab by default.

Reply:
Gotcha. FYI: I am seeing this in another setting where repo2docker is used, so this might be (slightly) more general than Binder.

Reply:
That's odd :confused:. @jtpio @ibdafna do you know why this is needed?

Reply:
> This should not be needed as `jupyter labextension install bqplot` did build Jupyter Lab.

Indeed it looks like it should not be needed. And the 2 calls to `jupyter labextension install` could be grouped together.